src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,14 +1,27 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+// import { store } from '../../app/store';
+import { actions as filterActions } from '../../features/filter';
+import { Status } from '../../types/Status';
+import { useAppSelector } from '../../app/hooks';
 
 export const TodoFilter: React.FC = () => {
+  const dispatch = useDispatch();
+  const filter = useAppSelector(state => state.filter);
+
   return (
     <form
       className="field has-addons"
       onSubmit={event => event.preventDefault()}
     >
       <p className="control">
         <span className="select">
-          <select data-cy="statusSelect">
+          <select
+            data-cy="statusSelect"
+            onChange={(event) => {
```

Comment:
it's better to move such handlers into a variable above (like `handleSelect`, `handleChange`, `handleClick`)